README.md

```diff
@@ -120,6 +120,7 @@ As a prelude, see the help:
     --http-auth-password <http_auth_password>  Password to authenticate all HTTP requests.
     --http-tls-disable                         Disable TLS/SSL when crawling or downloading pages.
     -V, --no-verbose                           Make the program silent.
+    --ignore-robots                            Ignore robots.txt file.
```

Comment:
But why? The file extension part is not an information you need in the option ;) `--ignore-robots` should be enough to understand it IMHO.